src/main/scala/uk/gov/nationalarchives/tdr/api/model/consignment/ConsignmentReference.scala

```diff
@@ -0,0 +1,19 @@
+package uk.gov.nationalarchives.tdr.api.model.consignment
+
+import uk.gov.nationalarchives.oci.Alphabet.Alphabet
+import uk.gov.nationalarchives.oci.{Alphabet, BaseCoder, IncludedAlphabet}
+
+object ConsignmentReference {
+  val alphabet: Alphabet = Alphabet.loadAlphabet(Right(IncludedAlphabet.GCRb25)) match {
+    case Left(error) => throw new Exception(s"Theres been an error loading the encoding alphabet", error)
```

Comment:
Sorry, one last thing: missing apostrophe in "Theres".

You could just drop the "Theres been an" part of it anyway, and say something like: `Error loading the GCRb25 encoding alphabet`.

Reply:
Sure, will do! :+1:

Reply:
Is there a commit that still needs to be pushed? This message looks the same as before.

Reply:
Ah weird, I think GitHub had a bit of a tizzy, I pushed it yesterday but it's only showing up now. I'll need to fix some conflicts now anyway though.